.eslintrc.js

```diff
@@ -211,15 +212,7 @@ module.exports = {
 				definedTags: [ 'jest-environment' ],
 			},
 		],
-		'react-compiler/react-compiler': [
-			'error',
-			{
-				environment: {
-					enableTreatRefLikeIdentifiersAsRefs: true,
-					validateRefAccessDuringRender: false,
-				},
```

Comment:
Is there any way to keep these React compiler config directives with the `react-hooks/rules-of-hooks` package? I remember these made sense back in the day.